components/groups/modals/updateGroupModal.tsx

```
@@ -215,74 +220,82 @@ export function UpdateGroupModal({
+              (Number(votingPeriod?.seconds) || 0);
+            const originalSeconds = Number(maybeVotingPeriod?.seconds) || 0;
+            const hasVotingPeriodChange = totalSeconds !== originalSeconds;
+
+            return hasThresholdChange || hasVotingPeriodChange;
+          },
+          then: schema => schema.optional(),
+          otherwise: schema =>
+            schema.when(['authors', 'summary', 'forum', 'description'], {
+              is: (authors: string, summary: string, forum: string, description: string) =>
+                !authors && !summary && !forum && !description,
+              then: schema => schema.required('At least one metadata field is required'),
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)